glusterd2/commands/volumes/common.go

```diff
@@ -249,6 +251,159 @@ func storeVolume(c transaction.TxnCtx) error {
 	return nil
 }
 
+func txnPrepareBricks(c transaction.TxnCtx) error {
+	var req api.VolCreateReq
+	if err := c.Get("req", &req); err != nil {
+		c.Logger().WithError(err).WithField(
+			"key", "req").Debug("Failed to get key from store")
+		return err
+	}
+
+	if req.Size == 0 {
+		return nil
+	}
+
+	for _, sv := range req.Subvols {
+		for _, b := range sv.Bricks {
+			peerid := uuid.Parse(b.PeerID)
+			if peerid == nil {
+				// TODO: Peer ID Parse error
+			}
+
+			if !uuid.Equal(peerid, gdctx.MyUUID) {
+				continue
+			}
+
+			brickMountDir := path.Dir(b.Path)
+
+			// Create Mount directory
+			err := os.MkdirAll(brickMountDir, os.ModeDir|os.ModePerm)
+			if err != nil {
+				c.Logger().WithError(err).
+					WithField("path", brickMountDir).
+					Error("Failed to create brick mount directory")
+				return err
+			}
+
+			// Create Logical Volume
+			cmd := exec.Command("lvcreate",
+				"--poolmetadatasize", "5K",
+				"-c", "256K",
```

Comment:
For all such defaults, I think we should either keep them dynamic or we should declare constants.

Reply:
These are dummy values, I will work on these values while working on SnapshotFactor